testModule/settings.gradle.kts

```diff
@@ -1,23 +1,13 @@
-import com.github.javaparser.printer.concretesyntaxmodel.CsmElement.attribute
-import groovy.xml.dom.DOMCategory.attributes
-
-pluginManagement {
-	repositories {
-		google()
-		gradlePluginPortal()
-		mavenCentral()
-
-		mavenLocal()
-	}
-
-}
-
 buildscript {
-	repositories {
-		google()
-		gradlePluginPortal()
-		mavenCentral()
-
-		mavenLocal()
-	}
+    pluginManagement {
+        repositories {
+            gradlePluginPortal()
+            maven("https://repo.sleeping.town") {
```

Comment:
This is not an acceptable solution - the entire point of shadowing FlexVer was to avoid doing this!

Reply:
This is only due to using include build, published version includes FlexVer in jar